unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -597,32 +597,55 @@ findAll = find' "find.all" Input.FindLocalAndDeps
 findGlobal :: InputPattern
 findGlobal = find' "find.global" Input.FindGlobal
 
+findIn :: InputPattern
+findIn =
+  InputPattern
+    "find-in"
+    []
+    I.Visible
+    [("namespace", Required, namespaceArg)]
```

Comment:
Remember to list the optional args for the find queries here please :)

Reply:
Good catch thanks.